models/state.py

```diff
@@ -1,8 +1,21 @@
 #!/usr/bin/python3
 """ State Module for HBNB project """
-from models.base_model import BaseModel
+from models.base_model import BaseModel, Base
+from sqlalchemy import Column, String
+from sqlalchemy.orm import relationship
 
 
-class State(BaseModel):
+class State(BaseModel, Base):
     """ State class """
-    name = ""
+    __tablename__ = 'states'
+    name = Column(String(128), nullable=False)
+
+    cascade = "all, delete-orphan"
+    """When a state oblect is deleted, all lonked City objects are deleted"""
```

Comment:
Spelling Mistake in Comment, should be "object" and not "oblect"

Reply:
Done